CONTRIBUTING.md

```diff
@@ -4,13 +4,27 @@ This is a [Jazzband](https://jazzband.co/) project. By contributing you agree
 to abide by the [Contributor Code of Conduct](https://jazzband.co/about/conduct)
 and follow the [guidelines](https://jazzband.co/about/guidelines).
 
+## Getting started
+
+Install tox:
+
+    pip install tox
```

Comment:
Since it's not RST, we should probably use markdown + maybe mark it with a `console` lexer:
```suggestion
```console
$ python -m pip install tox --user
```\
```

Reply:
Although I'm not sure if this should dive into the details of using tools like pip and tox — maybe it's better to point to the respective docs rather than attempting to maintain copies of their guides?

Reply:
Markdown seems to understand the format I used though (I agree it's missing the console lexer though)

![image](https://user-images.githubusercontent.com/12172017/116482832-3a2c5500-a886-11eb-9f02-ddf9515664ba.png)